vasyl_varyshchuk/Week_1/task_3/generate_file_size.py

```diff
@@ -0,0 +1,50 @@
+"""
+This module returns random size of file
+"""
+
+import random
+
+list_of_files = ['reports.txt', 'monthly_reports.zip', 'README.txt',
+                 'picture.jpg', 'logo.ttf', 'config.ini']
+
+
+def recognize_file_type(a_file):
+    return a_file.split('.')[1]
+
+
+def run_executors(file_name):
+    def txt_handler(file_ext):
```

Comment:
There is no sense to have an extension as a parameter.

Reply:
I agree. It can be used without a parameter.

Reply:
Appropriate changes were added in last commit.